frontend/src/app/projects/project-detail/project-users/project-user-settings.component.ts

```diff
@@ -170,7 +53,7 @@ export class ProjectUserSettingsComponent implements OnInit, OnChanges {
     this.projectUserService
       .deleteUserFromProject(this.project.slug, user.id, reason)
       .subscribe(() => {
-        this.refreshProjectUsers();
+        this.projectUserService.loadProjectUsers(this.project!);
```

Comment:
Similar to the `addUser` function, I would also move the `loadProjectUsers` to the `changeRoleOfProjectUser` function. As I can see we have several `this.projectUserService.loadProjectsUsers(...)` in this component which I would all move to the projectsUsersService

Reply:
Done